db/dcrpg/sync.go

```diff
@@ -113,11 +114,25 @@ func (db *ChainDB) SyncChainDB(client rpcutils.MasterBlockGetter, quit chan stru
 		db.EnableDuplicateCheckOnInsert(true)
 	}
 
-	// Add the various updates that should run on successful sync.
-	explorer.SyncStatusUpdate(0, 0, 0, dbtypes.InitialDBLoad, InitialLoadSyncStatusMsg)
-	// Addresses table sync should only run if bulk update is enabled.
-	if updateAllAddresses {
-		explorer.SyncStatusUpdate(0, 0, 0, dbtypes.AddressesTableSync, AddressesSyncStatusMsg)
+	if barLoad != nil {
+		// Add the various updates that should run on successful sync.
+		barLoad <- &dbtypes.ProgressBarLoad{
+			From:      0,
+			To:        0,
+			Timestamp: 0,
```

Comment:
I would elide all the zero value items (From, To, Timestamp) as this is done implicitly.